frontend/src/components/common/input/input.tsx

```diff
@@ -62,7 +68,16 @@ const Input: FC<Props> = ({
       >
         {label}
       </span>
-      {inputArea}
+      {isPassword ? (
+        <PasswordInput
+          control={control}
+          name={name}
+          inputClassName={styles.input}
+          placeholder="Enter your password"
+        />
+      ) : (
+        inputArea
+      )}
```

Comment:
```suggestion
```
you don't need to change the base input at all. all changes should be in the `PasswordInput`